src/NzbDrone.Core/Datastore/Migration/074_add_add_options_to_series.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(74)]
+    public class add_add_options_to_series : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("Series").AddColumn("AddOptions").AsString().Nullable();
```

Comment:
So we're going to add a column in this PR and then remove it again once the TaskQueue is implemented?